contracts/extensions/token/erc20/ERC20TokenExtension.sol

```diff
@@ -171,6 +171,20 @@ contract ERC20Extension is AdapterGuard, IExtension, IERC20 {
         return bank.balanceOf(account, tokenAddress);
     }
 
+    /**
+     * @dev Returns the amount of tokens owned by `account` considering the snapshot.
+     */
+    function getPriorAmount(address account, uint256 snapshot)
+        public
+        view
+        returns (uint256)
+    {
+        BankExtension bank = BankExtension(
```

Comment:
just a small optimisation, but it costs to use a local variable. I would just do

```
        return BankExtension(dao.getExtensionAddress(DaoHelper.BANK))
                    .getPriorAmount(account, tokenAddress, snapshot);
```